homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -117,6 +118,7 @@ def extra_state_attributes(self):
             ATTR_PLATFORM: self._opendata.connections[0]["platform"],
             ATTR_TRANSFERS: self._opendata.connections[0]["transfers"],
             ATTR_DURATION: self._opendata.connections[0]["duration"],
+            ATTR_DEPARTURE_TIME0: self._opendata.connections[0]["departure"],
```

Comment:
The departure time is already available as the main state. We should not duplicate that as a state attribute.

Reply:
I have no idea how I missed that. Apologies for taking up your time.